### PR TITLE
register yellorit ore & improve check for TileEntities in DeepslateOreRegistry

### PR DESCRIPTION
**Pull request by Pilzinsel64**

### CODE REVIEW

File: src/main/java/ganymedes01/etfuturum/recipes/ModRecipes.java [hunk 1]
```diff
@@ -998,6 +998,12 @@ ore_dyes[dye], new ItemStack(Blocks.sand, 1, 0), new ItemStack(Blocks.sand, 1, 0
 			}
 		}
 
+		if (Loader.isModLoaded("BigReactors")) {
```
Comment (Roadhog360) [inline src/main/java/ganymedes01/etfuturum/recipes/ModRecipes.java:1001]: Needs to use [ModsList](https://github.com/Roadhog360/Et-Futurum-Requiem/blob/master/src/main/java/ganymedes01/etfuturum/compat/ModsList.java)

**Review by Roadhog360 — CHANGES_REQUESTED** [review]

**Comment by Pilzinsel64** [comment]
Not sure if you like the way I choosen for checking for TileEntity. Either ask me to remove it again or change it after merging.

File: src/main/java/ganymedes01/etfuturum/api/DeepslateOreRegistry.java [hunk 2]
```diff
@@ -57,7 +57,7 @@ public static void addOre(Block from, Block to) {
 	 * @param toMeta   The meta deepslate changes it to
 	 */
 	public static void addOre(Block from, int fromMeta, Block to, int toMeta) {
-		if (from instanceof ITileEntityProvider || to instanceof ITileEntityProvider) {
+		if (from.hasTileEntity(fromMeta) || to.hasTileEntity(toMeta)) {
```
Comment (Roadhog360) [inline src/main/java/ganymedes01/etfuturum/api/DeepslateOreRegistry.java:60]: Revert this; I will add it later but I need to test this to ensure it does not have any additional undesired side-effects

**Review by Roadhog360 — CHANGES_REQUESTED** [review]

File: src/main/java/ganymedes01/etfuturum/api/DeepslateOreRegistry.java (hunk 2 above)
Comment (Pilzinsel64) [reply src/main/java/ganymedes01/etfuturum/api/DeepslateOreRegistry.java]: Yes Sir, it's reverted. 😊

**Review by Roadhog360 — APPROVED** [review]
Good!

**Pull request merged**